DEV - 1/FibonacciNumbers/FibonacciNumbers.sln

```diff
@@ -3,18 +3,18 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
```

Comment:
Very strange solution structure, it seems that you create new solution. Use the same solution, only add new project to it